lstchain/reco/volume_reducer.py

```diff
@@ -0,0 +1,145 @@
+"""
+**TEMPORTAL** module with functions to perform a volume reduction to LST data:
+
+NOTE : The volume reduction SHOULD BE performed at the dl0 data levels.
+
+This is a temporal implementation of the 'tailcut and dilation' volume reduction method that computes
+the mask from the dl1 (integrated image) level, and applies it to the image (dl1) as well as to the waveform (dl0) and
+pulse_time arrays (dl1), filling the 'anti-mask' with zeros.
+
+This means that the 'first' dl0 computed data is OVERWRITTEN by the vol. reduced dl0 data.
+
+Usage:
+
+"import volume_reducer"
+
+In the configuration file:
+
+(...)
+
+"volume_reducer":{
+    "algorithm": "zero_suppression_tailcut_dilation",
+    "parameters": { # add here the desired tailcut parameters. See function's help for default parameters.
+    }
+ }
+
+(...)
+"""
+from ctapipe.image.cleaning import tailcuts_clean, dilate
+
+__all__ = ['check_volume_reduction_method',
+           'apply_volume_reduction',
+           'zero_suppression_tailcut_dilation'
+           ]
+
+
+def check_volume_reduction_method(config_file):
+    """
+    Checks in the configuration file if a volume reduction method has been set.
+
+    Parameters
+    ----------
+    config_file: dict
+        The configuration file used in the parent code.
+
+    Returns
+    -------
+    flag: bool
+        Flag indicating if the volume reducer method declared in the configuration file is(not) none
+    algorithm: str
+        Volume reduction algorithm name
+    """
+    if config_file['volume_reducer']['algorithm'] is None:
+        flag = False
+        algorithm = ''
+    else:
+        flag = True
+        algorithm = config_file['volume_reducer']['algorithm']
+        print(algorithm)
+
+    return flag, algorithm
```

Comment:
This doubles the number of output with no clear benefit imo, since you will need to check the flag anyway afterwards instead of just checking if the algorithm is `None`